packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -848,6 +943,12 @@ export class Function extends FunctionBase {
       resource.tracingConfig = this.buildTracingConfig(props.tracing ?? Tracing.ACTIVE);
     }
 
+    this.loggingConfig = this.getLoggingConfig(props);
```

Comment:
why not just add `loggingConfig` directly to the `CfnLambda` resource getting created a few lines above?